dist/components/progress-steps/_progress-steps.scss

```diff
@@ -0,0 +1,108 @@
+// Progress
+// ========
+//
+// Helps show users where they exist within a multiple-page workflow, such as a
+// checkout flow.
+
+// Configurable Variables
+// ---
+
+$progress-steps__item-bg-color: hsl(0, 0%, 95%) !default;
+$progress-steps__item-padding: if(variable-exists(v-space), $v-space, 10px) !default;
+$progress-steps__layer: if(variable-exists(stage-layer), $stage-layer, 10) !default;
+$progress-steps__current-bg-color: darken($progress-steps__item-bg-color, 10%) !default;
+$progress-steps__completed-bg-color: darken($progress-steps__item-bg-color, 20%) !default;
+
+// Arrow Divider Variables
+// 1. These two variables together determine the angle of the arrow dividers
```

Comment:
There should be an additional space above this ordered list to prevent markdown from interpreting these two lines as a single paragraph.